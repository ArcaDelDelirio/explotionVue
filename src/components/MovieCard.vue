<script>
export default {
    name:'movieCard',
    props:{
        title: {
            type: String,
            required: true,
        }
    },

}
</script>

<template>
    <div class="card">
        
    </div>
</template>

<style scoped>
.card{
    display: flex,
}
</style>
