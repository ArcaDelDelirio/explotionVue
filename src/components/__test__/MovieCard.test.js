import { shallowMount } from '@vue/test-utils';
import MovieCard from '../MovieCard';

describe('MovieCard View', () => {
  it('Render the component', () => {
    const wrapper = shallowMount(MovieCard, 'title');
    expect(wrapper.html()).toMatchSnapshot();
  });
});
