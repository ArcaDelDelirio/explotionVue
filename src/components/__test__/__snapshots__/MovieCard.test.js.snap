// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard View Render the component 1`] = `<div class="card"></div>`;
